doc/conf.py

```diff
@@ -11,7 +11,7 @@
 # All configuration values have a default; values that are commented out
 # serve to show the default.
 
-import sys, os
+import sys, os, subprocess
```

Comment:
I dont see why this is needed -> maybe a leftover from testing?

Reply:
plantuml path uses this (missed it in a previous PR)